### PR TITLE
Remove all beta features from Issuing APIs

### PR DESCRIPTION
**Pull request by remi-stripe**
This PR removes all the deprecated features from the Issuing API and renames all relevant classes to ensure we have the right structure for codegen.

r? @richardm-stripe @ob-stripe 
cc @stripe/api-libraries 

---

Note: Targets major version integration branch in #1055.

### CODE REVIEW

File: charge.go [hunk 1]
```diff
@@ -485,7 +485,7 @@ type Charge struct {
 	OnBehalfOf                *Account                    `json:"on_behalf_of"`
 	Outcome                   *ChargeOutcome              `json:"outcome"`
 	Paid                      bool                        `json:"paid"`
-	PaymentIntent             string                      `json:"payment_intent"`
+	PaymentIntent             *PaymentIntent              `json:"payment_intent"`
```
Comment (remi-stripe) [inline charge.go]: Like last time, doing this in case we make it expandable (highly likely)

File: issuing_authorization.go [hunk 2]
```diff
@@ -235,7 +177,7 @@ type IssuingAuthorization struct {
 	Livemode            bool                                    `json:"livemode"`
 	MerchantAmount      int64                                   `json:"merchant_amount"`
 	MerchantCurrency    Currency                                `json:"merchant_currency"`
-	MerchantData        *IssuingMerchantData                    `json:"merchant_data"`
+	MerchantData        *IssuingAuthoriationMerchantData        `json:"merchant_data"`
```
Comment (ob-stripe) [inline issuing_authorization.go]: Typo in the name: `IssuingAuthoriationMerchantData` (missing a `z`)

Comment (remi-stripe) [reply issuing_authorization.go]: damn nice catch. Fixed PTAL

**Comment by brandur-stripe** [comment]
@remi-stripe Quick note that I retargeted this PR to the major integration branch in #1055.

File: issuing_cardholder.go [hunk 3]
```diff
@@ -139,24 +126,25 @@ type IssuingCardholderListParams struct {
...
-type IssuingBilling struct {
+// IssuingCardholderBilling is the resource representing the billing hash with the Issuing APIs.
+type IssuingCardholderBilling struct {
 	Address *Address `json:"address"`
+}
 
-	// This property is deprecated
-	Name string `json:"name"`
+// IssuingCardholderCompany represents additional information about a
+// business_entity cardholder.
```
Comment (richardm-stripe) [inline issuing_cardholder.go:137]: s/business_entity/company in the docstring, too

File: issuing_cardholder.go [hunk 4]
```diff
@@ -46,17 +43,11 @@ type IssuingCardholderType string
...
-type IssuingBillingParams struct {
+// IssuingCardholderBillingParams is the set of parameters that can be used for billing with the Issuing APIs.
+type IssuingCardholderBillingParams struct {
 	Address *AddressParams `form:"address"`
-
-	// This parameter is deprecated
-	Name *string `form:"name"`
 }
 
 // IssuingCardholderCompanyParams represents additional information about a
```
Comment (richardm-stripe) [inline issuing_cardholder.go:53]: this docstring also needs s/business_entity/company now

**Comment by remi-stripe** [comment]
Fixed the comments issues so approving and merging

**Pull request merged**